lib/src/includes/signed_video_sign.h

```diff
@@ -499,6 +499,29 @@ signed_video_set_recurrence_interval_frames(signed_video_t *self, unsigned recur
 SignedVideoReturnCode
 signed_video_set_sei_epb(signed_video_t *self, bool sei_epb);
 
+/**
+ * @brief Configures Signed Video use golden SEI principle
+ *
+ * The principle of the golden SEI sends all information only needed once, such as the Public key,
+ * at the start of the stream with the golden SEI. After that, the rest of the SEIs in the stream
+ * only include mandatory information.
+ *
+ * It is the user's responsibility to ensure that the first SEI in the stream is a golden SEI.
+ * This golden SEI does not necessarily have to be at the very beginning of the stream, but it must
+ * be the first SEI included.
+ *
+ * Note: It is not feasible to set this on an ongoing session.
+ *
+ * @param self Session struct pointer
+ * @param using_golden_sei Flag to enable or disable the golden SEI principle.
+ *
+ * @returns SV_OK SEI |using_golden_sei| was successfully set,
+ *          SV_INVALID_PARAMETER Invalid parameter.
+ *          SV_NOT_SUPPORTED if set on the ongoing session.
```

Comment:
Is there a test for verifying that you get SV_NOT_SUPPORTED if set during a session?

Reply:
I can add new tests in the next pull request